File: Python/PRAK504-2210817310001-MuhammadRyanRizkyRamadi.py
def reverse(rever):
    i = 0
    while rever != 0:
        i = (10 * i) + rever % 10
        rever //= 10
    return i

for i in range(3):
    nilai1, nilai2 = input().split()
    nilai1 = reverse(int(nilai1))
    nilai2 = reverse(int(nilai2))
    hasil = nilai1 + nilai2
    print(f'{reverse(hasil)}\n')

File: C/PRAK504-2210817310001-MuhammadRyanRizkyRamadi.c
#include <stdio.h>
int reverse(int rever) {
    int i = 0;
    while(rever != 0) {
        i = i*10;
        i = i+rever%10;
        rever/=10;
    }
    return i;
}

int main() {
    int A, B,i =0;
        scanf("%d %d",&A,&B); 
        A = reverse(A); 
        B = reverse(B);
        int C= A+ B; 
        printf("%d",reverse(C));
    return 0;
}
